loop/routes/rooms.js

```diff
@@ -386,16 +386,19 @@ module.exports = function (apiRouter, conf, logError, storage, filestorage, auth
                     if (res.serverError(err)) return;
 
                     // Room participants are used by metrics
+                    var otherParticipants = participants.filter(function(participant) {
```

Comment:
We should state what "otherParticipants" meants. It's the pariticpants which haven't joined yet.